Decred Wallet/Features/Security/SecurityViewController.swift

```diff
@@ -48,6 +48,8 @@ class SecurityViewController: SecurityBaseViewController {
     override func viewDidLoad() {
         // delay before activating initial tab to allow borders show properly
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+            self.btnPassword.setBackgroundColor(UIColor.appColors.transparentThinGray, for: .highlighted)
+            self.btnPin.setBackgroundColor(UIColor.appColors.transparentThinGray, for: .highlighted)
```

Comment:
The 2 lines of code above (for setting the background color) do not need to be inside this `asyncAfter` block. The comment on line 49 explains that the `asyncAfter` block is only necessary to handle correct border placement on the buttons. It's very safe to set background colors on the buttons as soon as the view loads.